### DISCUSSION TITLE
How to improve the speed of opening image

### DISCUSSION THREAD

**Discussion by jueqing1015** [dlemstra/Magick.NET#892] [Help]
Sometimes it takes a long time to open an image. I use the same image

FileStream pFileStream = new FileStream(Url, FileMode.Open , FileAccess.Read );

It's much faster than that MAGICK.NET It's much faster. Is there any way to improve its opening speed?

**Comment by dlemstra** [comment]
The reason that `FileStream` is much faster is because it doesn't need to decode the image.
